concent_api/core/message_handlers.py

```diff
@@ -489,30 +487,30 @@ def handle_send_force_subtask_results(client_message: message.concents.ForceSubt
             force_subtask_results=client_message,
             reason=message.concents.ForceSubtaskResultsRejected.REASON.RequestPremature,
         )
-    else:
-        subtask = store_or_update_subtask(
-            task_id                     = task_to_compute.compute_task_def['task_id'],
-            subtask_id                  = task_to_compute.compute_task_def['subtask_id'],
-            provider_public_key         = provider_public_key,
-            requestor_public_key        = requestor_public_key,
-            state                       = Subtask.SubtaskState.FORCING_ACCEPTANCE,
-            next_deadline               = forcing_acceptance_deadline + settings.CONCENT_MESSAGING_TIME,
-            set_next_deadline           = True,
-            ack_report_computed_task    = client_message.ack_report_computed_task,
-            task_to_compute             = client_message.ack_report_computed_task.report_computed_task.task_to_compute,
-            report_computed_task        = client_message.ack_report_computed_task.report_computed_task,
-        )
-        store_pending_message(
-            response_type       = PendingResponse.ResponseType.ForceSubtaskResults,
-            client_public_key   = requestor_public_key,
-            queue               = PendingResponse.Queue.Receive,
-            subtask             = subtask,
-        )
-        logging.log_message_added_to_queue(
-            client_message,
-            provider_public_key,
-        )
-        return HttpResponse("", status = 202)
+
+    subtask = store_or_update_subtask(
+        task_id                     = task_to_compute.compute_task_def['task_id'],
+        subtask_id                  = task_to_compute.compute_task_def['subtask_id'],
+        provider_public_key         = provider_public_key,
+        requestor_public_key        = requestor_public_key,
+        state                       = Subtask.SubtaskState.FORCING_ACCEPTANCE,
+        next_deadline               = forcing_acceptance_deadline + settings.CONCENT_MESSAGING_TIME,
+        set_next_deadline           = True,
+        ack_report_computed_task    = client_message.ack_report_computed_task,
+        task_to_compute             = client_message.ack_report_computed_task.report_computed_task.task_to_compute,
```

Comment:
`task_to_compute` could be extracted as variable nad reused

Reply:
:+1: done